react/features/unsupported-browser/components/__tests__/HideNotificationBarStyle.spec.js

```diff
@@ -0,0 +1,26 @@
+/* @flow */
+
+import { render } from 'enzyme';
+import React from 'react';
+
+import HideNotificationBarStyle from '../HideNotificationBarStyle';
+
+declare var describe: Function;
+declare var expect: Function;
+declare var it: Function;
+
+/**
+ * Test suite related to HideNotificationBarStyle component.
+ */
+describe('HideNotificationBarStyle component', () => {
+
+    /**
+     * HideNotificationBarStyle should render style element hiding notification
+     * bar.
+     */
+    it('should render style element', () => {
+        const wrapper = render(<HideNotificationBarStyle />);
+
+        expect(wrapper.find('style')).toHaveLength(1);
```

Comment:
Sure, this component doesn't do much but in order to cover it by tests I found this behavior the only thing that should be checked.